Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -183,6 +183,44 @@ import BraintreeDataCollector
         }
     }
 
+    /// Tokenize a PayPal request to be used with the Edit FI flow.
+    ///
+    /// On success, you will receive an instance of `BTPayPalAccountNonce`; on failure or user cancelation you will receive an error.
+    /// If the user cancels out of the flow, the error code will be `.canceled`.
+    ///
+    /// - Parameters:
+    ///   - request: A `BTPayPalEditRequest`
+    ///   - completion: This completion will be invoked exactly once when tokenization is complete or an error occurs.
+    /// - Warning: This feature is currently in beta and may change or be removed in future releases.
+    public func tokenize(
```

Comment:
Similar to Jax's comment re: only supporting new features in Swift, what do folks think about only supporting new features in the async-await method syntax? We could hold on the completion style syntax and add it later based on merchant feedback? Is now a good time to trial this?

Would love folks thoughts

Reply:
People I've talked to are working with async-await syntax in their code bases, but that's just a small sampling. I'm sure there are a lot of companies with legacy repos.

Reply:
that's a great point - i wonder how many merchants are transitioning to or already using the async/await style over the completion style syntax. imo i like that having both signatures gives merchants the choice to choose what works best for them but i also feel the asyn/await reads more clearly

Reply:
I think V7 will be a good chance to yank the completion syntax 👀